hadoop-master1/Dockerfile

```diff
@@ -36,6 +37,8 @@ ENV KYUUBI_HOME=/opt/kyuubi
 ENV KYUUBI_CONF_DIR=/etc/kyuubi/conf
 ENV RANGER_HOME=/opt/ranger
 ENV RANGER_VERSION=${RANGER_VERSION}
+ENV HUDI_HOME=/opt/hudi
```

Comment:
inline it to avoid potential side-effects, we don't need such env var

Reply:
> inline it to avoid potential side-effects, we don't need such env var

OK~